src/main/java/woohakdong/server/api/controller/member/MemberControllerDocs.java

```diff
@@ -0,0 +1,23 @@
+package woohakdong.server.api.controller.member;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import woohakdong.server.api.controller.member.dto.CreateMemberRequest;
+import woohakdong.server.api.controller.member.dto.MemberInfoResponse;
+
+@Tag(name = "Member", description = "멤버 관련 API")
+public interface MemberControllerDocs {
+
+    @SecurityRequirement(name = "accessToken")
+    @Operation(summary = "멤버 상세 정보 작성", description = "동아리원, 동아리 임원진 멤버 상세 정보 작성")
+    @ApiResponse(responseCode = "200", description = "멤버 상세 정보 작성 성공", useReturnTypeSchema = true)
+    public void createMember(CreateMemberRequest createMemberRequest);
```

Comment:
우선 이전에 Valid 어노테이션 없어도 되는 줄 알았는데, 테스트 해보니 착각이었나봅니다ㅎㅎ..
그리고 아래처럼 이렇게 Docs 쪽에다가 Valid 어노테이션 추가해도 동작하니, 참고하시면 좋을 것 같습니다
```suggestion
    public void createMember(@Valid CreateMemberRequest createMemberRequest);
```